packages/slider/src/mwc-slider-base.ts

```diff
@@ -18,11 +18,12 @@ import {applyPassive} from '@material/dom/events.js';
 import {addHasRemoveClass, EventType, FormElement, observer, SpecificEventListener} from '@material/mwc-base/form-element.js';
 import {MDCSliderAdapter} from '@material/slider/adapter.js';
 import MDCSliderFoundation from '@material/slider/foundation.js';
-import {html, property, query, TemplateResult} from 'lit-element';
+import {eventOptions, html, property, query, TemplateResult} from 'lit-element';
 import {classMap} from 'lit-html/directives/class-map';
 import {styleMap} from 'lit-html/directives/style-map';
 
-const {INPUT_EVENT, CHANGE_EVENT} = MDCSliderFoundation.strings;
```

Comment:
What happened here? Why can't we use the MDC constants?

Reply:
we don't use the MDC constants because they are prefixed `MDCSlider:change` or `MDCSlider:input` which are leaky and unnecessary since the element affords mdc slider

Reply:
For drawer we use MDCDrawer prefixed events: https://github.com/material-components/material-components-web-components/tree/master/packages/drawer#events -- why is this different?

> which are leaky and unnecessary since the element affords mdc slider

Sorry can you rephrase? Not quite sure I understand what this means.

Reply:
Oops I thought we did not use prefixed events for Drawer.

I think we should use `input` and `change` for slider though to emulate `<input type="range">`.

And by leaky, I mean that we are leaking the implementation details of this slider by firing an event that would not emulate the expected events from native `input`. The user would have to know about our internal implementation just to listen to a change and input event.

This is why we got rid of the prefixes in [dialog](https://github.com/material-components/material-components-web-components/tree/master/packages/dialog#events)